src/test/storage/encoded_column_test.cpp

```diff
@@ -159,7 +159,14 @@ TEST_P(EncodedColumnTest, SequentiallyReadNullableIntColumn) {
 
     value_column_iterable.with_iterators([&](auto value_column_it, auto value_column_end) {
       encoded_column_iterable.with_iterators([&](auto encoded_column_it, auto encoded_column_end) {
-        for (; encoded_column_it != encoded_column_end; ++encoded_column_it, ++value_column_it) {
+        auto i = 0;
```

Comment:
Can't see how these additions improve what `EXPECT_EQ(value_column_it->is_null(), encoded_column_it->is_null());` already checked, pls point me to it.
Also, variable name `i`....

Reply:
This covers `FrameOfReferenceColumn<T, U>::operator[]` which was [not covered before](https://ares.epic.hpi.uni-potsdam.de/jenkins/job/Hyrise/job/hyrise/job/mrks%252Fsome_tests/8//artifact/coverage_diff.html#src/lib/storage/frame_of_reference_column.cpp). I added a comment.

Not sure what to do about `i` - I don't want to call it `chunk_offset`, because in this context, there is not chunk. We could do `pos`, is that better?

Reply:
Just row_idx?

Reply:
anythings better than i

Reply:
ok